src/org/thoughtcrime/securesms/database/RecipientDatabase.java

```diff
@@ -50,11 +50,12 @@
   private static final String SIGNAL_PROFILE_NAME     = "signal_profile_name";
   private static final String SIGNAL_PROFILE_AVATAR   = "signal_profile_avatar";
   private static final String PROFILE_SHARING         = "profile_sharing_approval";
+  private static final String NICK_NAME               = "nick_name";
```

Comment:
nickname

Reply:
sure

Reply:
but the db attribute is set to nick_name, is that ok ?